malaria-app-android/src/main/java/com/peacecorps/malaria/ui/play/PlayFragment.java

```diff
@@ -98,4 +101,56 @@ public void onDetach() {
         listener = null;
     }
 
+    //Todo Kept for reference for showing dialog for playing the games, need to discuss in work session
```

Comment:
This is not recommended, even if you want to include/exclude or discuss, commented code  (dead code)should be removed. you can keep it offline somewhere for your reference and include once you are sure to include.